docs/pages/repo/docs/troubleshooting.mdx

```diff
@@ -80,6 +80,18 @@ directions even if you aren't planning to make a contribution.
 Turborepo does not support nested workspaces. Therefore, if you have a workspace that contains another workspace,
 you may see unexpected behavior. We recommend flattening your workspace structure to avoid this issue.
 
+### Root Dependencies
+
+If you have a package that is used by several other packages in the monorepo, say a standard ESLint configuration,
```

Comment:
```suggestion
If you have an internal package that is used by several other packages in the monorepo, say a standard ESLint configuration,
```

Reply:
I might try to phrase it without "internal" because that term isn't really explained until later on

Reply:
Agreed.